src/main/js/helpers/region.js

```diff
@@ -208,14 +208,51 @@ const shouldHideAllRegions = (regionList, graphTargets) =>
  * @returns {boolean} true if displayed targets is equal to 1, false otherwise
  */
 const isSingleTargetDisplayed = (graphTargets) => graphTargets.length === 1;
+/**
+ * Check all the regions within a graph is same or not
+ * @private
+ * @param {d3.selection} canvasSVG - d3 selection node of canvas svg
+ * @returns {Object} d3 svg path
```

Comment:
You are not returning anything in this function.

Reply:
ok will change it to **@returns {undefined} - returns nothing**